### DISCUSSION TITLE
如何主动向指定的群发送消息

### DISCUSSION THREAD

**Discussion by TwoOnefour** [nonebot/discussions#129] [问答] [answered]
我有个从wss服务器获取推送消息的需求，然后在服务器推送消息后主动向群里发送消息，请问该如何实现呢？

**Comment by yanyongyu** [comment, marked as answer]
可以通过调用api的方式主动向指定对象发送消息，参考[文档](https://v2.nonebot.dev/docs/next/tutorial/call-api)

如果你使用的是onebot adapter，参考[api文档](https://github.com/botuniverse/onebot-11/blob/master/api/public.md#send_group_msg-%E5%8F%91%E9%80%81%E7%BE%A4%E6%B6%88%E6%81%AF)

**Reply by TwoOnefour** [reply]
感谢回答。这里的文档我已经看过了，但是初学有不理解的地方，`bot = get_bot("bot_id")` 这里是该在插件里获取bot_id，还是应该再写个python文件呢

**Reply by yanyongyu** [reply]
get_bot只是获取bot的一种途径，在事件响应处理中就不需要通过get_bot来获取

**Reply by TwoOnefour** [reply]
我明白了，之前我get_bot()获取不到bot是因为在bot创建之前我就调用了，感谢指导
